src/renderer/processor.ts

```diff
@@ -151,8 +152,8 @@ export function getTypeForFile(logFile: UnzippedFile): LogType {
     return LogType.NETLOG;
   } else if (fileName.startsWith('ShipIt') || fileName.includes('SquirrelSetup')) {
     return LogType.INSTALLER;
-  } else if (fileName.includes('Default_logs') || /\w{9,}_\w{9,}_\d{16,}\.txt/.test(fileName)) {
-    return LogType.IOS;
+  } else if (fileName.includes('Default_logs') || fileName.includes('attachment') || /\w{9,}_\w{9,}_\d{16,}\.txt/.test(fileName)) {

```

Comment:
come to think of it, this should be fileName.startswith('attachment'), not includes